examples/transient/transient_ex3/claw_system.C

```diff
@@ -0,0 +1,971 @@
+
+  const MeshBase & mesh = this->get_mesh();
+  const unsigned int mesh_dim = mesh.mesh_dimension();
+
+  // Construct face FE objects for both "current" and "neighbor" Elems
+  FEType fe_type = variable_type(0);
+  std::unique_ptr<FEBase> fe_elem_face = FEBase::build(mesh_dim, fe_type);
+  std::unique_ptr<FEBase> fe_neighbor_face = FEBase::build(mesh_dim, fe_type);
+
+  // Quadrature rules for numerical integration.
+  QGauss qface(mesh_dim-1, fe_type.default_quadrature_order());
+  fe_elem_face->attach_quadrature_rule(&qface);
+  fe_neighbor_face->attach_quadrature_rule(&qface);
+
+  // Here we define some references to cell-specific data that
+  // will be used to assemble the linear system.
+  // Data for surface integrals on the element boundary
+  const std::vector<std::vector<Real>> & phi_face = fe_elem_face->get_phi();
+  const std::vector<Real> & JxW_face = fe_elem_face->get_JxW();
+  const std::vector<Point> & qface_normals = fe_elem_face->get_normals();
+  const std::vector<Point> & qface_points = fe_elem_face->get_xyz();
+
+  // Data for surface integrals on the neighbor boundary
+  const std::vector<std::vector<Real>> & phi_neighbor_face = fe_neighbor_face->get_phi();
+
+  // Data structures to contain the element and neighbor boundary matrix
+  // contribution. This matrices will do the coupling beetwen the dofs of
+  // the element and those of his neighbors.
+  // Ken: matrix coupling elem and neighbor dofs
+  DenseMatrix<Number> Kne;
+  DenseMatrix<Number> Ken;
+  DenseMatrix<Number> Kee;
+  DenseMatrix<Number> Knn;
+
+  // DofMap reference and vectors used for storing dof_indices
+  const DofMap & dof_map = get_dof_map();
+  std::vector<unsigned int> dof_indices;
+  std::vector<unsigned int> neighbor_dof_indices;
+
+  for (const auto & elem : mesh.active_local_element_ptr_range())
+  {
+    dof_map.dof_indices (elem, dof_indices);
+    const unsigned int n_dofs = dof_indices.size();
+
+    for (auto side : elem->side_index_range())
+    {
+      // Skip over boundary sides
+      if (elem->neighbor_ptr(side) != nullptr)
+      {
+        const Elem * neighbor = elem->neighbor_ptr(side);
+
+        // Get the global id of the element and the neighbor
+        const unsigned int elem_id = elem->id();
+        const unsigned int neighbor_id = neighbor->id();
+
+        // Compare element IDs since we don't want to compute the same contribution twice
+        if (elem_id < neighbor_id)
+        {
+          // Pointer to the element side
+          std::unique_ptr<const Elem> elem_side = elem->build_side_ptr(side);
+
+          // Reinitialize shape functions on the element side
+          fe_elem_face->reinit(elem, side);
+
+          // Find side QP locations in neighbor side reference space
+          std::vector<Point> qface_neighbor_points;
+          FEInterface::inverse_map(
+            elem->dim(),
+            fe_type,
+            neighbor,
+            qface_points,
+            qface_neighbor_points);
```

Comment:
Likewise for FEMap::inverse_map here

Reply:
Updated in [9e2be1a](https://github.com/libMesh/libmesh/pull/4073/commits/9e2be1a9f044314b2fed387eb57f7762a17a2b58)